dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -377,8 +377,8 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
       
       final String[] val = params.get("filter");
       if(val!=null && val[0].contains("Quality")) {
-        Browser browser = new UserAgent(req.getHeader("user-agent")).getBrowser();
-        if(browser == Browser.SAFARI || browser == Browser.MOBILE_SAFARI){
+        final Browser browser = new UserAgent(req.getHeader("user-agent")).getBrowser();
+        if(browser == Browser.SAFARI || browser.toString().contains("MOBILE")){

```

Comment:
Does android / android tablets support webp?  If so, we should support them.  Maybe something like:
```
        UserAgent userAgent = new UserAgent(req.getHeader("user-agent"));
        if(userAgent.getBrowser() == Browser.SAFARI ||  userAgent.getOperatingSystem() == OperatingSystem.IOS);
```

Reply:
done